diff --git a/drivers/net/ifc/base/ifcvf.h b/drivers/net/ifc/base/ifcvf.h
index 9be2770fea..e75d936eca 100644
--- a/drivers/net/ifc/base/ifcvf.h
+++ b/drivers/net/ifc/base/ifcvf.h
@@ -13,7 +13,7 @@
 #define IFCVF_SUBSYS_DEVICE_ID	0x001A
 
 #define IFCVF_MAX_QUEUES		1
-#define VIRTIO_F_IOMMU_PLATFORM		33
+// #define VIRTIO_F_IOMMU_PLATFORM		33
 
 /* Common configuration */
 #define IFCVF_PCI_CAP_COMMON_CFG	1
